src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -281,12 +280,22 @@ describe('findBreakingChanges', () => {
       }
     `);
 
-    expect(findBreakingChanges(oldSchema, newSchema)).to.deep.equal([
+    expect(findSchemaChanges(oldSchema, newSchema)).to.deep.equal([
```

Comment:
@wtrocki Since you effectively merging tests for `findBreakingChanges` and `findDangerousChanges` please adjust tests accordingly.
For example, rename this one to `should detect if a field is added to an input type` and remove the duplicating test:
https://github.com/graphql/graphql-js/blob/06ef5db069cf2f896c51126973c6100a33e1ea7c/src/utilities/__tests__/findBreakingChanges-test.js#L1026-L1047

Reply:
@wtrocki Please do the same for other tests where you modify snapshot.